app/Livewire/Server/Charts.php

```
@@ -28,6 +32,27 @@ public function mount(string $server_uuid)
     }
 
+    public function toggleMetrics()
+    {
+        try {
+            $this->authorize('update', $this->server);
+            $this->server->settings->is_metrics_enabled = ! $this->server->settings->is_metrics_enabled;
+            $this->server->settings->save();
+            $this->server->refresh();
+
+            if ($this->server->isMetricsEnabled()) {
+                StartSentinel::run($this->server, true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Actions\Server\StartSentinel' in method 'toggleMetrics'. (undefined)

(StaticAccess)